<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderManageDialog</class>
 <widget class="QDialog" name="OrderManageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    border: 1px solid #000000;
}

QWidget#toolbar
{
	background:#A0666666;
	border:none;
}
QLabel#lbTitle
{
	font-family:&quot;微软雅黑&quot;;
	color:#666666;
	font-size:14px;
}
QPushButton
{
	border:none;
}

QPushButton#btnMenu
{
	image:url(:/Images/menu_normal.png);
}
QPushButton#btnMenu:hover
{
	image:url(:/Images/menu_hover.png);
}
QPushButton#btnBack
{
	image:url(:/Images/back_normal.png);
}
QPushButton#btnBack:hover
{
	image:url(:/Images/back_hover.png);
}
QWidget#titlepanel
{
	background:#FFFFFF;
}

QProgressBar
{
	border:none;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="shadowbg" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="shadowinnerbg" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="TitlebarWidget" name="titlebar" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>28</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="titlepanel" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>32</height>
            </size>
           </property>
           <widget class="QLabel" name="lbTitle">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>54</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QPushButton" name="btnMenu">
            <property name="geometry">
             <rect>
              <x>470</x>
              <y>10</y>
              <width>22</width>
              <height>28</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>22</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>28</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QPushButton" name="btnBack">
            <property name="geometry">
             <rect>
              <x>170</x>
              <y>10</y>
              <width>22</width>
              <height>28</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>22</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>28</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QProgressBar" name="pbLoad">
            <property name="geometry">
             <rect>
              <x>200</x>
              <y>0</y>
              <width>118</width>
              <height>2</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>2</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>2</height>
             </size>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="webview" name="webView" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitlebarWidget</class>
   <extends>QWidget</extends>
   <header location="global">TitlebarWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>webview</class>
   <extends>QWidget</extends>
   <header>webview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="localman.qrc"/>
 </resources>
 <connections/>
</ui>
